archive.py

```diff
@@ -123,6 +132,14 @@ def get_html_directory():
 
     return settings.html_directory
 
+def check_config_file_present():
+    config_file =  Path("_config.yml")
```

Comment:
So we don't want `_config.yml` to be inside of the archive/ dir. It has to be in the repo root. where the style.css is located. But with this `Path("_config.yml")` it's not clear where it is located at? What if you run `archive.py` from your home dir instead of the repo root, wouldn't it be problematic?

Reply:
Should we force the user to run `archive.py` at repo root only? The other relative paths in the code won't work if we run the `archive.py` from anywhere else. The `json` directory and the `streams.yaml` file location are both relative to the repo root.
@rht

Reply:
> Should we force the user to run archive.py at repo root only? 

As in checking if the location of the user when running archive.py matches the repo root (directory containing archive.py) and then raise error if they don't match? Sure, in a separate PR I suppose. I'd expect most FOSS communities to just do it like Julia does where archive.py is run by a GitHub Actions script already provided in this repo. And the script does run archive.py from the repo.